src/skycoin/skycoin.go

```diff
@@ -315,6 +315,7 @@ func (c *Coin) configureDaemon() daemon.Config {
 	dc.Daemon.OutgoingMax = c.config.Node.MaxOutgoingConnections
 	dc.Daemon.DataDirectory = c.config.Node.DataDirectory
 	dc.Daemon.LogPings = !c.config.Node.DisablePingPong
+	dc.Daemon.BlockchainPubkey = cipher.MustPubKeyFromHex(c.config.Node.BlockchainPubkeyStr)
```

Comment:
Config.Node doesn't parse BlockchainPubkeyStr to BlockchainPubkey during its initialization?